Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.19
+
+COPY . /repocontents/
```

Comment:
The files we need are:
./v2/
/.*.json  minus package*.json, tsconfig.json, .mocharc.json, .eslintrc.json, 
./pack/*.json
./translations/*

and technically we don't need any of the .git folders under there, just json.

Reply:
done